Modules/_multiprocessing/multiprocessing.c

```diff
@@ -180,23 +161,23 @@ PyInit__multiprocessing(void)
         else
             py_sem_value_max = PyLong_FromLong(SEM_VALUE_MAX);
         if (py_sem_value_max == NULL)
-            return NULL;
+            return -1;
         PyDict_SetItemString(_PyMp_SemLockType.tp_dict, "SEM_VALUE_MAX",
                              py_sem_value_max);
     }
     PyModule_AddObject(module, "SemLock", (PyObject*)&_PyMp_SemLockType);
 #endif
 
     /* Add configuration macros */
-    temp = PyDict_New();
+    PyObject *temp = PyDict_New();
     if (!temp)
-        return NULL;
+        return -1;
 
 #define ADD_FLAG(name)                                            \
     value = Py_BuildValue("i", name);                             \
-    if (value == NULL) { Py_DECREF(temp); return NULL; }          \
+    if (value == NULL) { Py_DECREF(temp); return -1; }            \
     if (PyDict_SetItemString(temp, #name, value) < 0) {           \
-        Py_DECREF(temp); Py_DECREF(value); return NULL; }         \
+        Py_DECREF(temp); Py_DECREF(value); return -1; }           \
     Py_DECREF(value)
```

Comment:
I suggest to reformat the macro like this:

```
#define ADD_FLAG(name)                                        \
    do {                                                      \
        PyObject *value = PyLong_FromLong(name);              \
        if (value == NULL) {                                  \
            Py_DECREF(flags);                                 \
            return -1;                                        \
        }                                                     \
        if (PyDict_SetItemString(flags, #name, value) < 0) {  \
            Py_DECREF(flags);                                 \
            Py_DECREF(value);                                 \
            return -1;                                        \
        }                                                     \
        Py_DECREF(value);                                     \
    } while (0)
```

I also replaced Py_BuildValue() with PyLong_FromLong(). `PyObject *value = NULL;` above must be removed in this case.